lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -188,6 +189,20 @@ class DetailsRenderer {
     return element;
   }
 
+  /**
+   * @param {string} type
+   * @param {*} value
+   */
+  _renderUnknown(type, value) {
+    // eslint-disable-next-line no-console
+    console.error(`Unknown valueType: ${type}`, value);
+    const element = this._dom.createElement('details', 'lh-unknown');
+    this._dom.createChildOf(element, 'summary').textContent =
+      `Details type '${type}' unrecognized by this version of the report renderer.`;
```

Comment:
this should be on `Util.UIStrings`, probably?

Reply:
It can be if we modify it to not use ICU. If we want to add a {type} we need someway of jamming i18n into the report. #9166

I suggest leaving this not translated, add a TODO referring to #7238, and then we can handle it in a follow up that allows ICU in the report.